x-pack/plugins/ml/server/models/data_recognizer/modules/metrics_ui_hosts/manifest.json

```diff
@@ -34,5 +34,6 @@
       "file": "datafeed_hosts_network_out.json",
       "job_id": "hosts_network_out"
     }
-  ]
+  ],
+  "tags": []
```

Comment:
I think this should be ["observability"]

Reply:
Updated in https://github.com/elastic/kibana/pull/168721/commits/ca79700393667389ca8e418f44990977db8817c3